core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -225,6 +229,12 @@ export class ContributionsAndReview
           details.topic_name + ' / ' + details.story_title +
           ' / ' + details.chapter_title);
       }
+      const translationLengthLabelRequired = (
+        this.isReviewTranslationsTab() && this.activeExplorationId);
+
+      const translationLengthInWords = (
+        translationLengthLabelRequired ? this.computeTranslationLengthLabel(
+          suggestion.exploration_content_html) : undefined);
```

Comment:
This logic looks simple enough to just inline below instead of declaring extra variables.

Reply:
Done